live-build/variants/internal-dct/ansible/playbook.yml

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2024 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+- hosts: all
+  gather_facts: no
+  vars:
+    ansible_python_interpreter: /usr/bin/python3
+  roles:
+    - appliance-build.minimal-common
+    - appliance-build.minimal-internal
+    - appliance-build.virtualization-common
```

Comment:
Nit: Also, since this is an internal variant, do we want to run `virtualization-development` for developer conveniences like checking out some dev repos, including maybe `apigw-services`? @eyalkaspi-delphix thoughts?

Reply:
we could, I thinking I'd leave that up to the DCT team to decide.. it can be easy to add later

Reply:
Adding `apigw-services` wouldn't be very useful at this point (we'd need a different version of Java and other tooling to be able to build DCT on the engine directly).

Other dev repos seem useful though, I guess this is necessary to be able to run `git appliance-update` for instance on such engines?